nntrainer/layers/preprocess_translate_layer.h

```diff
@@ -64,27 +64,29 @@ class PreprocessTranslateLayer : public Layer {
    * @retval #ML_ERROR_NONE Successful.
    * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
    */
-  int initialize(Manager &manager);
+  int initialize(Manager &manager) override;
 
   /**
    * @copydoc Layer::forwarding()
    */
-  void forwarding();
+  void forwarding(bool training = true) override;
```

Comment:
is it right?? training = true?

Reply:
Yes, semantic is defaulted to be training = true when just do `forwarding()` for every layer.

For augmentation layer, there is no difference.

Reply:
What I mean.. what it means if the training = true for translate layer. does it update weight?? or others?

Reply:
As per below comment, when training is false, there is no augmentation happening. when training is true, augmentation take places.

> For augmentation layer, their forwarding must be no_op (`hidden_ = input_`) when `training = false`.
> Please update